pkgs/applications/audio/spotify/darwin.nix

```diff
@@ -0,0 +1,28 @@
+{ fetchurl, stdenv, lib, undmg }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.42.151.g19de0aa6";
+  name = "spotify-mac";
+
+  src = fetchurl {
+    url = "http://download.spotify.com/Spotify.dmg";
```

Comment:
This is what they link to on the website (but with a versioned file)
https://d1clcicqv97n4s.cloudfront.net/Spotify-1.0.42.151.g19de0aa6.dmg